app/scripts/modules/ecs/src/serverGroup/configure/wizard/capacityProvider/CapacityProvider.tsx

```diff
@@ -37,10 +65,10 @@ class EcsCapacityProvider extends React.Component<IEcsCapacityProviderProps, IEc
     this.setState({capacityProviderStrategy : capacityProviderStrategy });
   }
 
-  private updateCapacityProviderName = (index: number, targetCapacityProviderName: string) => {
+  private updateCapacityProviderName = (index: number, targetCapacityProviderName: any) => {
     const capacityProviderStrategy = this.state.capacityProviderStrategy;
     const targetCapacityProviderStrategy = capacityProviderStrategy[index];
-    targetCapacityProviderStrategy.capacityProvider = targetCapacityProviderName;
+    targetCapacityProviderStrategy.capacityProvider = targetCapacityProviderName.label;
```

Comment:
if you pass in the label as the input, you wouldn't have to get the label.

Reply:
Yes, the change has been made.